### ISSUE TITLE
Hyperlink not readable on Program Expectations page

### ISSUE THREAD

**Issue by jomarieminney** [WordPress/five-for-the-future#208]
On this page: [Program Expectations](https://wordpress.org/five-for-the-future/expectations/) the hyperlink at the top is dark blue, on a dark blue background. It is slightly better but still very difficult to read on hover.
Screenshot: https://www.screencast.com/t/og30o4M5lS1

**Comment by costdev** [comment]
Looks like [this block](https://github.com/WordPress/five-for-the-future/blob/production/themes/wporg-5ftf/css/utilities/_colors.scss#L5-L7) may need to be adjusted to something like:

```scss
.has-wporg-blue-background-color {
    background-color: $color__wporg-blue;

    &.has-wporg-white-color a {
        color: $color__text-on-dark;
    }
}
```

This should produce results like the header of [The Patterns Directory](https://wordpress.org/patterns/), which styles `.site-link` with `color: #fff;`.

**Comment by pkevan** [comment]
Looks like this was updated with an inline style. 

closes #209
